src/plugins/guided_onboarding/kibana.json

```diff
@@ -6,13 +6,10 @@
     "name": "Journey Onboarding",
     "githubTeam": "platform-onboarding"
   },
-  "description": "Guided Onboarding",
+  "description": "Guided onboarding framework",
   "server": true,
   "ui": true,
-  "requiredBundles": [
-    "kibanaReact"
-  ],
-  "requiredPlugins": ["navigation"],
```

Comment:
AFAICT we are not using the `navigation` plugin. It was also causing the following error when I made `guidedOnboarding` a required plugin for the `home` plugin:

```
Topological ordering of plugins did not complete, these plugins have cyclic or missing dependencies
```